dashboard/app/controllers/registrations_controller.rb

```diff
@@ -76,6 +76,11 @@ def new_sign_up
     render 'new_sign_up'
   end
 
+  # Part of the new sign up flow - work in progress
+  def account_type
+    view_options(full_width: true, responsive_content: true)
```

Comment:
TIL about these options! Are they needed to make sure the page is responsive?

Reply:
Yeah! This felt like a monumental find ✨